clang/lib/DPCT/Error.cpp

```diff
@@ -191,13 +191,13 @@ void ShowStatus(int Status, std::string Message) {
   case CallIndependentToolError:
     StatusString = "Error: Call to " + Message + " failed.";
     break;
-  case MigrationErrorCMakeScriptPathInvalid:
-    StatusString = "Error: Path of CMake Script is invalid.";
+  case MigrationErrorCMakeAndPythonSetupScriptPathInvalid:
```

Comment:
I think we don't need update update the enum to contains all kinds of build script,  if we add one more, t hen it needs to update again.  How about give me neutral name? like (MigrationErrorbuildScriptPathInvalid)